src/model/webdav/WebdavFileSystem.ts

```diff
@@ -129,5 +129,5 @@ function extractData<T>(res: T | ResponseDataDetailed<T>): T {
 }
 /** Check if res is detailed response. */
 function isDetailedData(res: unknown | ResponseDataDetailed<unknown>): res is ResponseDataDetailed<unknown> {
-	return (res as ResponseDataDetailed<unknown>).data !== undefined;
+	return (res as ResponseDataDetailed<unknown>)?.data !== undefined;
```

Comment:
I feel like this should be handled in `extractData` and thrown as an `Error` there. The response being null is an error and I don't want `extractData` to return null.

https://github.com/club-1/webdav-drive/blob/48a0d72129885ad2bfc3661578edd378be42e839/src/model/webdav/WebdavFileSystem.ts#L123-L129

And then add a catch block to the `getQuota` call in App.js : https://github.com/club-1/webdav-drive/blob/48a0d72129885ad2bfc3661578edd378be42e839/src/App.svelte#L74

I imagine something like this:

```js
$: fs?.getQuota()
    .then((a) => (quota = a))
    .catch(() => (quota = null))
    ?? (quota = null);
```

The catch there would also probably be enough to solve your issue.